### PR TITLE
update key transactions with starred

### PR DESCRIPTION
**Pull request by andriisoldatenko**

### CODE REVIEW

**Review by priscilawebdev — APPROVED** [review]
lgtm

File: src/docs/product/data-management-settings/dynamic-sampling/index.mdx [hunk 1]
```diff
@@ -44,9 +44,9 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
-### Key Transactions
+### Starred Transactions
 
-Typically, for most projects there are only a few transactions that you really care about. We prioritize [key transactions](/product/performance/transaction-summary/#starring-key-transactions) over others, so you can have more visibility into your important data.
+Typically, for most projects there are only a few transactions that you really care about. We prioritize [starred transactions](/product/performance/transaction-summary/#starring-key-transactions) over others, so you can have more visibility into your important data.
```
Comment (priscilawebdev) [inline src/docs/product/data-management-settings/dynamic-sampling/index.mdx:49]: it would be nice to update the screenshot too

Comment (imatwawana) [reply src/docs/product/data-management-settings/dynamic-sampling/index.mdx:49]: Agreed, we'll want to update the screenshot before pushing this change.

**Review by imatwawana — COMMENTED** [review]
Once the screenshot is updated, this is good to go. I've created PR #5850 to move more towards the "starred" terminology.

**Review by imatwawana — APPROVED** [review]

**Pull request merged**